_data/library/jsfacades.yml

```diff
@@ -117,7 +117,7 @@
     - name: scalajs-angulate
       url: https://github.com/jokade/scalajs-angulate
       desc: Another binding to AngularJS with enhancements
-      dep: '"biz.enef" %%% "scalajs-angulate" % "0.2.2"'
+      dep: '"biz.enef" %%% "scalajs-angulate" % "latest.version"'
```

Comment:
Please use a fixed version number here. We don't want to encourage people to have non-reproducible builds.